applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -213,7 +232,7 @@ static int do_fota_start(int op, const char *file_uri, int sec_tag,
 	char *tmp = strrchr(path, '/');
 
 	if (tmp != NULL) {
-		char path2[SLM_MAX_URL] = { 0 };
+		char path2[FILE_URI_MAX] = { 0 };
 
```

Comment:
This is a bit risky, increase a lot in stack usage.
Make sure that the workqueue has enough stack space.

Reply:
I agree, this code will be changed later. There is no need to reserve that from stack but modify the original buffer. This code is inside "#ifdef CONFIG_SECURE_BOOT" and the code do not even compile right now.  There are related ticket for this task in Jira.